src/core/transport/transport.h

```diff
@@ -72,6 +72,12 @@ typedef struct grpc_transport_stream_op {
 
   grpc_stream_op_buffer *recv_ops;
   grpc_stream_state *recv_state;
+  /** The number of bytes this peer is currently prepared to receive.
+
+      Bytes offered are used to replenish per-stream flow control windows.
+      Offers are not retractable: if 5 bytes are offered and no bytes are read,
+        a later offer of 3 bytes still implies that 5 have been offered. */
```

Comment:
Done.

I haven't managed to repro the ASAN flake we saw earlier in 40k runs. Going to watch Jenkins this time and see if anything changes.